client/src/components/DataFiles/DataFilesSidebar/DataFilesSidebar.js

```diff
@@ -118,7 +118,13 @@ const DataFilesSidebar = ({ readOnly }) => {
                       activeClassName="active"
                     >
                       <div className="nav-content">
-                        <Icon name={sys.icon || 'my-data'} />
+                        <Icon
+                          name={
+                            sys.icon || sys.scheme === 'public'
+                              ? 'publications'
```

Comment:
this can actually be set in `settings_secret.py`! In the settings block `_PORTAL_DATAFILES_STORAGE_SYSTEMS`, set the `icon` for `Public Data` to `publications` like so:

```
_PORTAL_DATAFILES_STORAGE_SYSTEMS = [
    ...,
    {
        'name': 'Public Data',
        'system': 'cep.storage.public',
        'scheme': 'public',
        'api': 'tapis',
        'icon': 'publications'
    },
    ...
```

Could you make that addition to `settings_secret.example.py`?

Reply:
Thank you!